e2e/tests/regression/test_scheduling.robot

```diff
@@ -51,3 +52,43 @@ Test Soft Anti Affinity Scheduling
     Then Wait until volume 0 replicas rebuilding completed
     And Wait for volume 0 healthy
     And Check volume 0 data is intact
+
+Test Replica Auto Balance Node Least Effort
+    [Tags]    coretest
+    Given Set setting replica-soft-anti-affinity to true
+    And Set setting replica-auto-balance to least-effort
+
+    When Disable node 1 scheduling
+    And Disable node 2 scheduling
+    And Create volume 0 with    numberOfReplicas=6    dataEngine=${DATA_ENGINE}
+    And Attach volume 0
+    And Wait for volume 0 healthy
+    And Write data to volume 0
+    Then Volume 0 should have 6 replicas running on node 0
+    And Volume 0 should have 0 replicas running on node 1
+    And Volume 0 should have 0 replicas running on node 2
+
+    When Enable node 1 scheduling
+    # wait for auto balance
+    Then Volume 0 should have replicas running on node 1
+    And Volume 0 should have 6 replicas running
+    # loop 3 times with 5-second wait and compare the replica count to:
+    # ensure no additional scheduling occurs
+    # the replica count remains unchanged
+    And Volume 0 should have replicas running on node 0 and no additional scheduling occurs
+    And Volume 0 should have replicas running on node 1 and no additional scheduling occurs
+    And Volume 0 should have 0 replicas running on node 2 and no additional scheduling occurs
+
+    When Enable node 2 scheduling
+    # wait for auto balance
+    Then Volume 0 should have replicas running on node 2
+    And Volume 0 should have 6 replicas running
+    # loop 3 times with 5-second wait and compare the replica count to:
+    # ensure no additional scheduling occurs
+    # the replica count remains unchanged
+    And Volume 0 should have replicas running on node 0 and no additional scheduling occurs
+    And Volume 0 should have replicas running on node 1 and no additional scheduling occurs
+    And Volume 0 should have replicas running on node 2 and no additional scheduling occurs
```

Comment:
Same as above comment, including the replica count.

Reply:
We've already check
```
And Volume 0 should have 6 replicas running
```
in the previous step.

Reply:
Added.